### ISSUE TITLE
Export seekToSeconds method 

### ISSUE THREAD

**Issue by alejomendoza** [paramaggarwal/react-native-youtube#14]
I've been trying to do this but still I haven't figured out how to make it work.

on my RCTYouTubeManager.m:
```javascript
RCT_EXPORT_METHOD(seekTo:(NSString *)seconds) {
    
    float fseconds = [seconds floatValue];
    
   [[RCTYouTube alloc] seekToSeconds:fseconds allowSeekAhead:@YES];
    
}
```

on my react native file:
```javascript
 seekToSeconds(fractionValue){

        console.log(fractionValue);

        var seconds = fractionValue * this.state.duration;

        console.log(seconds);
        console.log(NativeModules);

        NativeModules.YouTubeManager.seekTo(seconds);

    }
```

**Comment by paramaggarwal** [comment]
In the manager, you have to pull out the view from the UI registry, and from React you'll have to pass the React tag.

```
RCT_EXPORT_METHOD(seekTo:(NSNumber *)reactTag seconds:(NSNumber *)seconds)
{
    [self.bridge.uiManager addUIBlock:
     ^(__unused RCTUIManager *uiManager, RCTSparseArray *viewRegistry){
         RCTYouTube *youtube = viewRegistry[reactTag];
         if ([youtube isKindOfClass:[RCTYouTube class]]) {
             [youtube seekToSeconds:seconds.floatValue allowSeekAhead:@YES];
         } else {
             RCTLogError(@"Cannot seek: %@ (tag #%@) is not RCTYouTube", youtube, reactTag);
         }
     }];
}
```

```
NativeModules.YouTubeManager.seekTo(
 React.findNodeHandle(this),
 5
);
```

**Comment by alejomendoza** [comment]
Ah thank you!! I was able to make it work, so I had a few errors at the beginning but I was able to solve: 

I had to change to this on RCTYouTubeManager your code to make it work:

```javascript
- (dispatch_queue_t)methodQueue
{
    return _bridge.uiManager.methodQueue;
}
```

Then I had to create a ref on my react youtube tag because I'm using a slider to trigger the seek functionality:

```javascript
 seekToSeconds(fractionValue){
        var seconds = fractionValue * this.state.duration;
        NativeModules.YouTubeManager.seekTo(
            React.findNodeHandle(this.refs.youtubeplayer),
            parseFloat(seconds)
        );
    },
    render(){
        return(
        <View  style={styles.container}>
            <NavBar downBtn={true} backBtn={false} fwdBtn={false} logoType={true} transparentBackground={false}/>
            <YouTube
                videoId={this.state.entryUid} // The YouTube video ID
                play={this.state.isPlaying}           // control playback of video with true/false
                hidden={false}        // control visiblity of the entire view
                playsInline={true}    // control whether the video should play inline
                onReady={(e)=>{this.setState({isReady: true})}}
                onChangeState={(e)=>{this.setState({status: e.state})}}
                onChangeQuality={(e)=>{this.setState({quality: e.quality})}}
                onError={(e)=>{this.setState({error: e.error})}}
                onPlayTime={(e)=>{this.setState({playTime: e.playTime, progress:e.playTime/e.duration, duration:e.duration})}}
                style={styles.youtubePlayer}
                ref="youtubeplayer"
                />
            <Slider
                value={this.state.progress}
                onValueChange={(value) => this.seekToSeconds(value)}
                 />
```

I still don't understand pretty well the UiManager stuff but I'll get there. (complete objective-c noob)

**Comment by paramaggarwal** [comment]
Yes, will keep both the issues open so these can be included in the project.

**Comment by paramaggarwal** [comment]
Merged in #28.
